halo2_proofs/src/dev/gates.rs

```diff
@@ -152,7 +154,10 @@ impl CircuitGates {
                         ),
                         queries: constraint.evaluate(
                             &|_| BTreeSet::default(),
-                            &|selector| vec![format!("S{}", selector.0)].into_iter().collect(),
+                            &|selector| iter::once(format!("S{}", selector.0)).collect(),
+                            &|virtual_col| {
+                                iter::once(format!("V{}", virtual_col.0.index())).collect()
+                            },
```

Comment:
Maybe not in this PR. But if we plan to support `Instance` cols too, we should change `V` for `virtual_col.0.col_type()`